src/keymaster-api.js

```diff
@@ -20,395 +20,425 @@ app.use(express.static(path.join(__dirname, 'kc-app/build')));
 
 v1router.get('/ready', async (req, res) => {
     try {
-        res.json('ready');
+        res.status(200).json('ready');
```

Comment:
Why specify when status(200) is the default?

Reply:
solid point! I added status(200) for extra clarity, but you're right, it's the default. I'll clean that up.